<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrthogonalSeriesWidget</class>
 <widget class="QMainWindow" name="OrthogonalSeriesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="originalTab">
       <attribute name="title">
        <string>Original</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="F2DViewerWidget" name="fvOriginal" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="realizationTab">
       <attribute name="title">
        <string>Realization</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="F2DViewerWidget" name="fvRealization" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="fitTab">
       <attribute name="title">
        <string>Fit</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="F2DViewerWidget" name="fvFit" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="xTab">
       <attribute name="title">
        <string>X</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="PlotView" name="xView" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="yTab">
       <attribute name="title">
        <string>Y</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="PlotView" name="yView" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>F2DViewerWidget</class>
   <extends>QWidget</extends>
   <header>F2DViewerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PlotView</class>
   <extends>QWidget</extends>
   <header>QatPlotWidgets/PlotView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
